buildmaster/CMS_TTBAR_8TEV_LJ_DIF/metadata.yaml

```diff
@@ -0,0 +1,113 @@
+# implemented by Tanishq Sharma
+
+setname: "CMS_TTBAR_8TEV_LJ_DIF"
+
+nnpdf_metadata:
+  nnpdf31_process: "TTB"
+  experiment: "CMS"
+
+arXiv:
+  url: "https://arxiv.org/abs/1505.04480"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1370682"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1370682"
+  version: 1
+
+version: 1
+version_comment: 'NA'
```

Comment:
This does not match [the request by @scarlehoff](https://github.com/NNPDF/nnpdf/pull/1684#issuecomment-1490177558)

Reply:
ok - sorry! you see I didn't check either :upside_down_face: maybe the comment can be improved still

Reply:
My fault, I should've updated the comment. I've done it now.

Reply:
even fine :upside_down_face: but I meant the `version_comment`

Reply:
@felixhekhorn , the way I see `version_comment`, it is to be used if there is something specific that needs to be mentioned for the specification. For e.g., if there was a bug in specific version, and you fix it in new version, you specify that correction, or if in the next version you implement for some more variables, you could specify that. so I do not think there is any compulsion for everyone to use "initial implementation" for every implementation. It is just a relic from the NMC prototype implementation. Same is the case for the version_number, the '0' was a relic of the NMC prototype. If an implementation if done successfully, it wouldn't make sense to call it version 0.